lib/system_description_store.rb

```diff
@@ -159,4 +130,20 @@ def get_backup_name(description_name)
 
     backup_name
   end
+
+  def system_description_exists!(description_name)
+    if !list.include?(description_name)
+      raise Machinery::Errors::SystemDescriptionNotFound.new(
+        "System description \"#{description_name}\" does not exist."
+      )
+    end
+  end
+
+  def system_description_exists?(description_name)
```

Comment:
To be consistent with other methods which do checks the method should probably rather named validate_nonexistence_of_description